software/duet_config_files/duet2/rrf3/sys/homey.g

```diff
@@ -4,4 +4,5 @@ G91                     ; Set relative mode
 G1 Y-386 F6000 H1       ; Big negative move to search for endstop
 G1 Y4 F600              ; Back off the endstop
 G1 Y-10 F600 H1         ; Find endstop again slowly
+G1 Y2 F600              ; Relieve strain from endstop
```

Comment:
These Y (and X and Z) [D2HW-XXXX](https://www.digikey.com/en/products/detail/omron-electronics-inc-emc-div/D2HW-C201H/1811903) endstops can actually be plunged beyond their trigger position by a substantial amount.
Since leaving this in does no harm, let's remove it to err on the side of simplicity so that folks reading this later don't wonder why this needed to be here.

Reply:
Relief line removed
Maximum search distance of 386 increased to 400. Homing from within the tool post zone can require more travel than previous maximum.